sdk/dapp-kit/src/hooks/wallet/useAutoConnectWallet.ts

```diff
@@ -1,51 +1,62 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
-import { useEffect } from 'react';
+import { useQuery } from '@tanstack/react-query';
 
 import { useConnectWallet } from './useConnectWallet.js';
 import { useCurrentWallet } from './useCurrentWallet.js';
 import { useWallets } from './useWallets.js';
 import { useWalletStore } from './useWalletStore.js';
 
 export function useAutoConnectWallet(autoConnectEnabled: boolean) {
```

Comment:
should this default to true?

Reply:
RE defaulting to true, I think we've decided at least for now to disable autoconnect to false, but would like to reconsider.